### ISSUE TITLE
Unable to train Chatterbot while using it with Django

### ISSUE THREAD

**Issue by Sinchana** [gunthercox/ChatterBot#717]
Hi,

I am creating a chatbot application using Django and Chatterbot and setting the basic static data in the "training_data" parameter,
CHATTERBOT = {
    'name': 'ChatterBot Example',
    'trainer':'chatterbot.trainers.ListTrainer',
	'logic_adapters': [	               
        'chatterbot.logic.BestMatch'
    ],
    'training_data': [ "How are you?","I am good.", "That is good to hear.","Thank you","You are welcome." ],	
    'django_app_name': 'django_chatterbot'
}

When i run the application in browser, not able to get the response which i am setting in training_data.
Can someone please let me know how to set the data here?

**Comment by vkosuri** [comment]
The reason you were not getting response from chatbot because you haven't trained your bot. Training bot from Django interface is different form regular chatterbot. 

``` Bash
python manage.py train
``` 
More information about Django training available here http://chatterbot.readthedocs.io/en/stable/django/training.html

**Comment by Sinchana** [comment]
@vkosuri : Thank you so much for the response,

The response is not as expected. Even after running the train command

Now i am training the python as mentioned "python manage.py train". The code mentioned below.

CHATTERBOT = {
    'name': 'Tech Support Bot',
    'logic_adapters': [       
        'chatterbot.logic.BestMatch'
    ],
    'trainer': 'chatterbot.trainers.ListTrainer',
    'training_data': [
         'Hello','hi there!'
    ]
}


Thanks in advance

**Comment by thedp** [comment]
After training with English greetings corps it can only respond to single words like: "Hello", "Hi", "Greetings!"

But it fails to reply to other phrases in that corps, like: "Hi, How is it going?"

**Comment by sporting** [comment]
Is this what you want? 
http://chatterbot.readthedocs.io/en/stable/logic/index.html#specific-response-adapter

**Comment by nonlin** [comment]
Training with the default settings.py doesn't seem to do anything for me as well. Still results in Hello responses.

**Comment by gunthercox** [comment]
I've labeled this as a possible bug. I will investigate this as soon as possible.

**Comment by zxxgobig** [comment]
@gunthercox hi,i run the python manage.py train but the response is not what i want .did this bug had fixed?

**Comment by ahmedshoaib** [comment]
using sql databases with django seems to be resulting in wrong training i.e responses mapping to wrong input statements

**Comment by gabru-md** [comment]
@ahmedshoaib are you suggesting that using a different database like mongodb could potentially solve this problem?

**Comment by ahmedshoaib** [comment]
@gabru-md  yes mongodb works perfectly fine with the django_app (I.e  with [use_django_models](https://github.com/gunthercox/ChatterBot/pull/653) : False). There seems to be a problem in storing training data in relational dbs.

**Comment by saraalosaid** [comment]
i have the same problem ,iam wondering how to use mongodb with the django

**Comment by gunthercox** [comment]
@saraalosaid If you want to use MongoDB and Django then you will need to configure configure Django's storage backend to use one the is build for MongDB. I highly recommend **Django MongoDB Engine** for this: https://django-mongodb-engine.readthedocs.io/en/latest/

**Comment by kunalwanikar** [comment]
@gabru-md  did it work using mongo DB?

**Comment by MARUTHESH** [comment]
I am getting this error 
FileNotFoundError: [Errno 2] No such file or directory: 'How are you?'
